packages/@lwc/ssr-compiler/src/compile-template/transformers/element.ts

```diff
@@ -235,6 +231,8 @@ export const Element: Transformer<IrElement | IrExternalComponent | IrSlot> = fu
         childContent = [];
     }
 
+    cxt.hoist(bImportHtmlEscape(), importHtmlEscapeKey);
+
```

Comment:
We can probably simplify our `ctx.hoist()` logic elsewhere as well (since we can basically just import everything from `@lwc/ssr-runtime` once and let the optimizer do its magic) but I've punted on that for now.